<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V800Main</class>
 <widget class="QWidget" name="V800Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V800 Downloader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="exerciseTree">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="downloadBtn">
         <property name="text">
          <string>Download
Sessions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dirSelectBtn">
         <property name="text">
          <string>Change Default
Output Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tcxBox">
         <property name="text">
          <string>TCX Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hrmBox">
         <property name="text">
          <string>HRM Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="gpxBox">
         <property name="text">
          <string>GPX Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkBtn">
         <property name="text">
          <string>Check All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uncheckBtn">
         <property name="text">
          <string>Uncheck All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fsBtn">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Browse 
V800...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
